lynx/common/square.py

```diff
@@ -12,21 +12,27 @@ class Square:
 
     def append(self, object: Object) -> NoReturn:
         # We might rename the walkable property into sth like ground
-        if object.walkable and self.ground != None:
+        if 'walkable' in object.tags and self.ground != None:
             # In the future when we add logger we should log this event!
             raise Exception("Cannot put more than a one ground in a square!")
-        
-        if object.walkable:
+
+        if 'walkable' in object.tags:
```

Comment:
`<tag(s)> in object.tags` is used a lot
I think that we really need `has_tags` method for `Object`